package.json

```diff
@@ -12,7 +12,7 @@
     "test:regression:approve": "yarn loki approve --port 9006",
     "test:regression:update": "yarn loki update --port 9006",
     "test:regression:update:storie": "yarn loki update --port 9006 --storiesFilter $STORIE",
-    "test:regression:storybook": "start-storybook -s static -c .storybook/regression-test --port 9006",
+    "test:regression:storybook": "fuser -k 9006/tcp; start-storybook -s static -c .storybook/regression-test --port 9006",
```

Comment:
Do you know why this alert prints here?
![image](https://user-images.githubusercontent.com/3389749/65177529-393a0d00-da2d-11e9-92de-5898e7982fa2.png)